l10n_eu_product_adr/models/product_dangerous.py

```diff
@@ -16,6 +18,11 @@ class ProductDangerousClass(models.Model):
         string="Dangerous Type",
     )
 
+    @api.depends("name", "code")
+    def _compute_complete_name(self):
+        for record in self:
+            record.complete_name = " ".join([record.code, record.name])
```

Comment:
What about adding the class_type_id representation in the complete_name ?

Reply:
Looking at the existing data, it seems to me that there would be a lot of duplicate information by adding the `class_type_id` name and not much gain.
The code is being added because from our experience people working with this data can quickly gather more information with the number in the code.